test/test_helper.exs

```diff
@@ -1 +1,6 @@
-ExUnit.start()
+# clang-format is required to format the generated code
+# it won't match the reference files otherwise
+case System.cmd("clang-format", ~w(--version)) do
```

Comment:
1. `System.cmd/1` raises `%ErlangError{orginal: :enoent}`, when it can't find a command
2. Why do you use one time `$ whereis clang-format` and second time `$ clang-format` to check, if there `clang-format` is available?
3. Analogically, why do you use `System.cmd/2` in one place, to execute command, but `Mix.shell().cmd/1" in another place?

Reply:
Honestly speaking I don't know what is the difference between `System.cmd` and `Mix.shell().cmd()` except that `Mix` might not be available in the runtime. I moved everything to `System`